### ISSUE TITLE
Different types of agents

### ISSUE THREAD

**Issue by falkogloeckler** [DINA-Web/agent-specs#18]
Agents are not necessarily only natural persons, but could also be corporate bodies (e.g. organizations).

Agents act as owner, collector, determinator, donor etc. in a collection.
It's quite usual that organizations are donors of specimens in the context of collection accessions.

Thus, the agents module need to address that by accommodating different types.

**Comment by cgendreau** [comment]
Can an organization be affiliated to another organization?

**Comment by falkogloeckler** [comment]
Yes! For example: MfN is a member institution of the Leibniz Association.

**Comment by dshorthouse** [comment]
And organizations do not persist. And they can have different, time-dependent names. And they have names in different languages. You see where this is going...
